### ISSUE TITLE
work on startStopTable lost on last merge?

### ISSUE THREAD

**Issue by davidjamesbeck** [paul-shannon/slexil#44]
Hi, Paul

Now that I'm going back over text.py and ijaLine.py, I see that these have been reverted to the older versions that required a startStopTable as a parameter fed to the Text constructor. It looks like when you merged the startStopTalbe branch, git replaced the modified files with the versions you had on the master branch. (They may have been newer, but the modifications we wanted were on startStopTable.) Is there some way to restore those files?

David

**Comment by paul-shannon** [comment]
Hi David,

Here below is what I see on the master branch for the text.py constructor.  Are you also on the master branch?  I don't see a startStopTable parameter in the ctor!

````bash> git branch -v````

![image](https://user-images.githubusercontent.com/2480712/61175685-7e8f1980-a568-11e9-962b-e80666a1264b.png)

**Comment by davidjamesbeck** [comment]
I seem to be on master, 

iMac-2:slexil David$ git branch -v
  DownloadPackage     8b5a2be [ahead 10] Merge branch 'Order_of_operations'
  Order_of_operations 8b5a2be [ahead 1] Merge branch 'Order_of_operations'
  TierMapping         8b5a2be [ahead 4] Merge branch 'Order_of_operations'
  TwoNewLines         8b5a2be [ahead 17] Merge branch 'Order_of_operations'
* master              01bef37 [behind 18] Merge remote-tracking branch 'origin/master'
  startStopTables     ff448ef Merge branch 'master' into startStopTables
  workflow_control    5bdcdb7 [ahead 1] playaudioinsequence

but it says “behind 18”. I wonder if I wasn’t on Master when I did my last pull yesterday. I don’t know why I would have been anywhere else … . Anyway, I’ll copy the files I modified and then do another pull. sigh.

David
